### ISSUE TITLE
Cant Use en locale

### ISSUE THREAD

**Issue by Tankonyako** [sokil/php-isocodes#56]
Hello im using your library, but i can't use en or en_US or en_GB languages, it giving, but it working with another localizations
```
   InvalidArgumentException 

  Directory does not contain valid resource

  at ****\vendor\sokil\php-isocodes\src\TranslationDriver\SymfonyTranslationDriver.php:46
     42▕             }
     43▕         }
     44▕
     45▕         if ($validPathToMoFile === null) {
  ➜  46▕             throw new \InvalidArgumentException('Directory does not contain valid resource');
     47▕         }
     48▕
     49▕         $this->translator->addResource(
     50▕             'mo',
Sokil\IsoCodes\TranslationDriver\SymfonyTranslationDriver::configureDirectory("3166-1", "****\vendor\sokil\php-isocodes\src/../../php-isocodes-db-i18n/messages")

```
My code:
```
$driver = new Sokil\IsoCodes\TranslationDriver\SymfonyTranslationDriver();
$driver->setLocale('en');
$__iso_codes = new \Sokil\IsoCodes\IsoCodesFactory(null, $driver);
$__iso_codes->getCountries()->getByAlpha2('UA');
		``

**Comment by sokil** [comment]
It's the case when mo file for related iso not found. Due to "en" as default language, there is no *.mo files at all. I'l make fix to return default (en) value if related mo file not found. wait for next release

І Слава Україні!

**Comment by Tankonyako** [comment]
я пофіксував це якщо en то використовується GetText translator

**Comment by sokil** [comment]
Fix present in version 4.1.0
